app/Jobs/CheckTraefikVersionJob.php

```
@@ -0,0 +1,225 @@
+            }
+
+            // Phase 2: Group by team and send notifications
+            Log::info('CheckTraefikVersionJob: Phase 2 - Grouping by team and sending notifications');
+
+            $serversByTeam = $outdatedServers->groupBy('team_id');
+            $teamCount = $serversByTeam->count();
+
+            Log::info("CheckTraefikVersionJob: Grouped outdated servers into {$teamCount} team(s)");
+
+            foreach ($serversByTeam as $teamId => $teamServers) {
+                $team = Team::find($teamId);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\Team' in method 'handle'. (undefined)

(StaticAccess)